Makefile

```diff
@@ -3,7 +3,7 @@
 
 BRANCH         := $(shell git rev-parse --abbrev-ref HEAD)
 COMMIT         := $(shell git log -1 --format='%H')
-BUILD_DIR      ?= $(CURDIR)/build
+BUILD_DIR      ?= $(CURDIR)/./build
```

Comment:
Is this because git was tracking it? We could just add build to the .gitignore. Not sure if @adamewozniak cares?

Reply:
just a personal preference, would revert it back to /build